drivers/stepper/ti/drv8424.c

```diff
@@ -9,7 +9,8 @@
 #include <zephyr/drivers/stepper.h>
 #include <zephyr/drivers/gpio.h>
 #include <zephyr/drivers/stepper/stepper_drv8424.h>
-#include "../step_dir/step_dir_stepper_common.h"
+// #include "../step_dir/step_dir_stepper_common.h"
+#include "../step_dir/step_dir_stepper_stm_timer.h"
```

Comment:
I think no relative paths shall be used. Can't find any rules from zephyr side - but I've not seen any relative include paths so far.